WalletWasabi/Models/CoinsRegistry.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using NBitcoin;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Gui.Models
+{
+	public class CoinsRegistry
+	{
+		private HashSet<SmartCoin> _coins;
```

Comment:
Always use properties.

Reply:
Why use a property if a private field can do the job?

Reply:
To be consistent with the rest of the codebase. Also it's the best practice: https://csharpindepth.com/articles/PropertiesMatter

Reply:
Thank you.
Note that the article is talking about `public property` vs `non-private field`, but in our case it is `private property` vs `private field`.
Take a look at this question for reasons to use private properties:
https://stackoverflow.com/questions/3310186/are-there-any-reasons-to-use-private-properties-in-c